client/lib/user/support-user-interop.js

```diff
@@ -18,27 +24,117 @@ import { getSupportUser, getSupportToken } from 'state/support/selectors';
  *
  * @param {Object}  reduxStore  The global redux store instance
  */
-export default function( reduxStore ) {
-	const user = new User();
+class SupportUser {
```

Comment:
We can maybe leave this for a future PR, but since this is basically a singleton, I think we may be able to work around needing a class. Maybe something like:
```javascript
const reduxStoreReadyPromise = new Promise( ( resolve ) => { 
    setReduxStore = ( store ) => resolve( store ); 
} );
const supportUserIsEnabled = config.isEnabled( 'support-user' );
const export fetchToken( user, password ) {
    if( supportUserIsEnabled ) { 
        return reduxStoreReadyPromise.then(
             //...
        );
    }
}
```